### PR TITLE
negative tests for ras authn

### PR DESCRIPTION
**Pull request by haraprasadj**

### CODE REVIEW

File: suites/apis/rasAuthN.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+ RAS Integration Test - AuthN Only (PXP-6505)
+ This test plan has a few pre-requisites:
+ 1. Fence > 4.23.0 must be deployed.
+ 2. There must be an upstream client previously registered with NIH/RAS and
+    the test environment must have its client & secret ID stored into its Fence config.
+    e.g., $ cat gen3_secrets_folder/configs/fence-config.yaml | yq .OPENID_CONNECT.ras
+          {
+              "client_id": "****",
+              "client_secret": "****",
+              "redirect_url": "{{BASE_URL}}/login/ras/callback"
```
Comment (themarcelor) [inline suites/apis/rasAuthN.js:12]: I guess we should update these comments to make clear that the `discovery_url` is now mandatory. 🤷

File: suites/apis/rasAuthN.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
+/* eslint-disable codeceptjs/no-skipped-tests */
```
Comment (themarcelor) [inline suites/apis/rasAuthN.js:1]: Do we want to have these new scenarios in a whole new/separate file?
`rasAuthN.js` or should we just append these scenarios to the existing `rasIntegrationTest.js` script?

**Review by themarcelor — APPROVED** [review]
+1
Left a few comments, but nothing critical.
Thanks 👍

File: suites/apis/rasAuthN.js (hunk 2 above)
Comment (haraprasadj) [reply suites/apis/rasAuthN.js:1]: These are idempotent tests ... we can merge back later by refactoring the existing tests to run independently in the same way. For now, I will leave it here.

**Pull request merged**